src/events.ts

```diff
@@ -119,7 +121,9 @@ export type SeamEvent =
   | FailedToSetOnDeviceAccessCodeEvent
   | DelayInSettingOnDeviceAccessCodeEvent
   | NoiseDetectedEvent
-  | ConnectedAccountConnected
-  | ConnectedAccountDisconnected
+  // | ConnectedAccountConnected
+  // | ConnectedAccountDisconnected
```

Comment:
intentionally commented out?

Reply:
whoops I have no idea how that snuck in, thanks